docs/development/local-keep-network.adoc

```diff
@@ -213,6 +213,10 @@ To run the script some manual preparation is needed:
 - <<client-config-file>> for the single client (default name: `config.toml`).
 
 === Install script
+First be sure you have created a `src` directory in `$HOME/go/src`
+for our go packages if it does not already exist. Then ensure you have
+`protoc-gen-gogoslick` already installed and your go `bin` is accessible from your path `/Users/username/go/bin`. To check run `which protoc-gen-gogoslick`.
```

Comment:
I was about to say this is probably not required and then I went to `go.mod` just to see we refer `1.12` there.
We'll solve it separately in another PR, heh.

Reply:
Can you please break the line here so that it respects 80-characters ruler?